client/cluster_profile.go

```diff
@@ -207,6 +208,41 @@ func (h *V1Client) GetClusterProfile(uid string) (*models.V1ClusterProfile, erro
 	return resp.Payload, nil
 }
 
+type values struct {
+	HeaderMap map[string]string
+}
+
+func (v values) Get(key string) string {
+	if v.HeaderMap == nil {
+		return ""
+	}
+	return v.HeaderMap[key]
+}
+
+// DownloadClusterProfileUIDSpc retrieves an existing cluster profile by UID.
+func (h *V1Client) DownloadClusterProfileUIDSpc(w *io.PipeWriter, profileUid, projectUid string) (string, error) {
+	type CustomHeader string
+
+	const CustomHeaders CustomHeader = "CustomHeaders"
```

Comment:
All of this project code should be removed. The client is already scoped to a project or tenant level.